app/controllers/api/base_controller/renderer.rb

```diff
@@ -165,6 +165,18 @@ def collection_search(is_subcollection, type, klass)
         Rbac.filtered(res, options)
       end
 
+      def virtual_attribute_search(resource, attribute)
+        if resource.class < ApplicationRecord
+          if resource.class.try(:plural?, attribute)
+            Rbac.filtered(resource.public_send(attribute))
+          else
+            Rbac.filtered_object(resource).try(:public_send, attribute)
```

Comment:
there are no RBAC check where associations are requested thru `attributes` params like /api/providers/2?attributes=parent_manager.host, parent_manager.cloud_tenants

we have RBAC check for [collections](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api/base_controller/renderer.rb#L165)  and as you said for [resource](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/api/base_controller/renderer.rb#L130) but not for the `?attributes`